### PR TITLE
fixed nav/header partials

### PR DESCRIPTION
**Pull request by erilun404**
fixade filer

### CODE REVIEW

File: views/pages/animals.ejs [hunk 1]
```diff
@@ -2,9 +2,7 @@
 <html lang="en">
 <%- include("../partials/head") %>
 <body>
-    <header>
-      <%- include("../partials/navigation")%>
-    </header>
+    <%- include("../partial/header") %>
```
Comment (davidboll) [inline views/pages/animals.ejs:5]: It should be partials, not partial

File: views/pages/birds.ejs [hunk 2]
```diff
@@ -2,18 +2,7 @@
...
-           <ul class="nav-menu">
-                   <li class="nav-item"><a href="/" class="nav-link">Home</a></li>
-                <li class="nav-item"><a href="/animals/mammals" class="nav-link">Mammals</a></li>
-                <li class="nav-item"><a href="/animals/reptiles"   class="nav-link">Reptiles</a></li>
-                <li class="nav-item"><a href="/animals/birds" class="nav-link">Birds</a></li>
-              </ul>
-            </header>
-          </div>
-    </header>
+   <%- include("../partial/header")  %>
```
Comment (davidboll) [inline views/pages/birds.ejs:5]: Same here :)

File: views/pages/index.ejs [hunk 3]
```diff
@@ -2,9 +2,7 @@
 <html lang="en">
 <%- include("../partials/head") %>
 <body>
-    <header>
-       <%- include("..partial/navigation")%>
-    </header>
+    <%- include("../partial/header")  %>
```
Comment (davidboll) [inline views/pages/index.ejs:5]: Same here :)

File: views/pages/reptiles.ejs [hunk 4]
```diff
@@ -2,18 +2,7 @@
...
-              <ul class="nav-menu">
-            <li class="nav-item"><a href="/" class="nav-link">Home</a></li>
-                <li class="nav-item"><a href="/animals/mammals" class="nav-link">Mammals</a></li>
-                <li class="nav-item"><a href="/animals/reptiles"   class="nav-link">Reptiles</a></li>
-                <li class="nav-item"><a href="/animals/birds" class="nav-link">Birds</a></li>
-              </ul>
-            </header>
-          </div>
-    </header>
+    <%- include("../partial/header")  %>
```
Comment (davidboll) [inline views/pages/reptiles.ejs:5]: Same here 👍

**Review by davidboll — CHANGES_REQUESTED** [review]
Nice job! Just need to add an s in partials :)

**Comment by MartinAhlsen** [comment]
Fint!

**Pull request merged**